windows-mixin/dashboards/windows_logs.libsonnet

```
@@ -4,16 +4,17 @@ local custom_barchart_grafonnet = import '../lib/custom-barchart-grafonnet/custo
 
 local host_matcher = 'job=~"$job", agent_hostname=~"$hostname"';
 local log_channel_matcher = host_matcher + ', channel=~"$channel"';
+local windows_event_parser = '| json | line_format "{{.execution_processId}} Source: {{.source}} EventID: {{.event_id}} Level: {{.levelText}} . {{.message}}"';
 
 local queries = {
   total_log_lines: 'sum(count_over_time({' + log_channel_matcher + '}[$__interval]))',
   total_log_warnings: 'sum(count_over_time({' + log_channel_matcher + '} |= "Warning" [$__interval]))',
   total_log_errors: 'sum(count_over_time({' + log_channel_matcher + '} |= "Error" [$__interval]))',
   error_percentage: 'sum( count_over_time({' + log_channel_matcher + '} |= "Error" [$__interval]) ) / sum( count_over_time({' + log_channel_matcher + '} [$__interval]) )',
```

```suggestion
  error_percentage: 'sum(count_over_time({' + log_channel_matcher + '} |= "Error" [$__interval]) ) / sum( count_over_time({' + log_channel_matcher + '} [$__interval]) )',
```
Remove to match other lines style?